packages/fether-react/src/i18n/index.js

```diff
@@ -0,0 +1,98 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// https://react.i18next.com/getting-started
+import i18next from 'i18next';
+// // https://github.com/i18next/i18next-browser-languageDetector
+import LanguageDetector from 'i18next-browser-languagedetector';
```

Comment:
Does this work in Electron?

Reply:
Yes, you're correct. I just removed it and it works, so we don't need it